src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -54,60 +55,61 @@
  */
 public class Drivetrain extends SubsystemBase {
 
-  private ShuffleboardTab m_swerveModulesTab, m_drivetrainTab;
-
-  // Odometry
-  private final SwerveDrivePoseEstimator m_poseEstimator;
-
   // This is left intentionally public
   public final Module[] m_modules;
 
   private final WPI_Pigeon2 m_pigeon;
-  private Vision m_vision;
+  private final Vision m_vision;
+  // Odometry
+  private final SwerveDrivePoseEstimator m_poseEstimator;
+  // Displays the field with the robots estimated pose on it
+  private final Field2d m_fieldDisplay;
+
+  private boolean m_visionEnabled = true;
+  private boolean m_chargeStationVision = false;
 
   // PID Controllers for chassis movement
   private final PIDController m_xController;
   private final PIDController m_yController;
   private final PIDController m_rotationController;
 
-  private boolean m_chargeStationVision = false;
-
-  // Displays the field with the robots estimated pose on it
-  private final Field2d m_fieldDisplay;
-
+  // PIDs used in PathPlanner command
   private final PIDController m_pathplannerXController;
   private final PIDController m_pathplannerYController;
   private final PIDController m_pathplannerRotationController;
 
-  //Shuffleboard
+  private final PIDController m_balancePID;
+
+  // Shuffleboard
+  private final ShuffleboardTab m_swerveModulesTab, m_drivetrainTab;
+  
   private GenericEntry 
     m_driveVelocityEntry,
-    m_steerVelocityEntry, 
+    m_steerVelocityEntry,
     m_steerAngleEntry,
-    m_driveStaticFeedforwardEntry, 
-    m_driveVelocityFeedforwardEntry, 
+    m_driveStaticFeedforwardEntry,
+    m_driveVelocityFeedforwardEntry,
     m_steerStaticFeedforwardEntry,
     m_steerVelocityFeedforwardEntry,
     m_xPosEntry,
     m_yPosEntry,
     m_headingEntry;
-  
+
   private Double[] m_driveVelFeedForwardSaver = new Double[4];
   private Double[] m_driveStaticFeedForwardSaver = new Double[4];
   private Double[] m_steerVelFeedForwardSaver = new Double[4];
   private Double[] m_steerStaticFeedForwardSaver = new Double[4];
-  
+
```

Comment:
As a great man once said, "Remove random spaces, I know it is knit picky but it is annoying for a few reasons. It make the file change view larger than it needs to be. It makes random files appear in the pull request. Random spaces can add up and It makes random gaps of code which increases the distance to scroll though code. Everyone dose it but at least clean it up before you make a PR. So I know it's knit picky but it makes is easier for the people reviewing and working on the code."